src/utilities/modules/vaccines.js

```diff
@@ -165,3 +165,87 @@ export const sensorLogsExtractBatches = ({ sensorLogs = [], itemBatch, item } =
     items: allItemsForLogs,
   };
 };
+
+export const aggregateLogs = sensorLogs => [
+  { timestamp: 'Feb 23', temperature: 5.9, sensorLog: sensorLogs[0] },
+  { timestamp: 'Feb 24', temperature: 6, sensorLog: sensorLogs[0] },
+  { timestamp: 'Feb 25', temperature: 7, sensorLog: sensorLogs[0] },
+  { timestamp: 'Feb 26', temperature: 8.4, sensorLog: sensorLogs[0] },
+  { timestamp: 'Feb 27', temperature: 8, sensorLog: sensorLogs[0] },
+  { timestamp: 'March 1', temperature: 6, sensorLog: sensorLogs[0] },
+  { timestamp: 'March 2', temperature: 5.7, sensorLog: sensorLogs[0] },
+  { timestamp: 'March 3', temperature: 3.5, sensorLog: sensorLogs[0] },
+  { timestamp: 'March 4', temperature: 3.8, sensorLog: sensorLogs[0] },
+  { timestamp: 'March 5', temperature: 2.2, sensorLog: sensorLogs[0] },
+];
+
+export const extractBreachPoints = ({ lineData, fullBreaches, temperatureRange }) => [
+  {
+    timestamp: 'Feb 26',
+    temperature: 8.4,
+    sensorLogs: [],
+    isMax: true, // sensorLogs.max('temperature') > temperatureRange.maxTemperature;
+    ...temperatureRange,
+  },
+];
+
+/**
+ * Returns aggregated data for breach modal, based on passed array
+ * of sensorLogs (breaches)
+ */
+export const extractDataForBreachModal = ({
```

Comment:
breaches in this method refers to output of extractBreachPoints. Would be good if it's generic for all use cases though.